src/net.cpp

```diff
@@ -2539,15 +2539,44 @@ std::vector<CAddress> CConnman::GetAddresses()
     return addresses;
 }
 
-std::vector<CAddress> CConnman::GetAddresses(Network requestor_network)
+const std::vector<CAddress>& CConnman::GetAddresses(Network requestor_network, CAddress local_socket_address)
```

Comment:
Is there any scenario where CAddress does not imply a Network?

Reply:
With Tor we may be seeing an IPv4 connection between Bitcoin Core and the Tor proxy (with IPv4 addresses) but the "requestor network" should be Tor, not IPv4.